### PR TITLE
[tmux/fr] Add french translation

### PR DESCRIPTION
**Pull request by martin-damien**

### CODE REVIEW

File: fr-fr/tmux-fr.html.markdown [hunk 1]
```diff
@@ -0,0 +1,256 @@
+---
+category: tool
+tool: tmux
+contributors:
+    - ["mdln", "https://github.com/mdln"]
+translators:
+    - ["@DamienVGN", "https://github.com/martin-damien"]
+lang: fr-fr
+filename: LearnTmux.txt
```
Comment (vendethiel) [inline fr-fr/tmux-fr.html.markdown]: `-fr.html`

File: fr-fr/tmux-fr.html.markdown [hunk 2]
```diff
@@ -0,0 +1,256 @@
+---
+category: tool
+tool: tmux
+contributors:
+    - ["mdln", "https://github.com/mdln"]
+translators:
+    - ["@DamienVGN", "https://github.com/martin-damien"]
```
Comment (vendethiel) [inline fr-fr/tmux-fr.html.markdown]: pas de `@`

File: fr-fr/tmux-fr.html.markdown [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+setw -g monitor-activity on
+set -g visual-activity on
+set-option -g bell-action any
+set-option -g visual-bell off
+
+# Définition automatique du titre des fenêtres
+set-option -g set-titles on
+set-option -g set-titles-string '#H:#S.#I.#P #W #T' # window number,program name,active (or not)
+
+# Ajustements de la barre de status
```
Comment (vendethiel) [inline fr-fr/tmux-fr.html.markdown]: Ajustement

File: fr-fr/tmux-fr.html.markdown [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+bind E swap-window -t -1
+bind F swap-window -t +1
+
+# Raccourcis pour scinder les zones
+bind = split-window -h
+bind - split-window -v
+unbind '"'
+unbind %
+
+# Définis une commande pour envoyer un préfixe dans un tmux qui serais appelé dans un tmux
```
Comment (vendethiel) [inline fr-fr/tmux-fr.html.markdown]: serait

**Comment by vendethiel** [comment]
Thanks & sorry for the delay! (we don't get notifications for commits..)

**Comment by aloisdg** [comment]
@martin-damien Any updates on this?

**Comment by martin-damien** [comment]
Yes everything was corrected. I haven't seen the notification for "awaiting response".

**Comment by vendethiel** [comment]
Yeah, we don't get notifications for commits either... It seems it has a few conflicts - do you have time to fix those?

**Comment by martin-damien** [comment]
I will not resolve the conflicts because an another translation have been accepted before mine and looks pretty correct. This translation is has no reason to be anymore.

That's the game :)

See you on an another PR.

**Pull request closed without merging**